lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -100,13 +117,12 @@ fb_actions.create = function(prompt_bufnr)
   local current_picker = action_state.get_current_picker(prompt_bufnr)
   local finder = current_picker.finder
 
-  local default = get_target_dir(finder) .. os_sep
-  vim.ui.input({ prompt = "Insert the file name: ", default = default, completion = "file" }, function(input)
+  local base_dir = get_target_dir(finder) .. os_sep
+  vim.ui.input({ prompt = "Insert the file name: ", default = base_dir, completion = "file" }, function(input)
     vim.cmd [[ redraw ]] -- redraw to clear out vim.ui.prompt to avoid hit-enter prompt
     local file = create(input, finder)
     if file then
-      -- values from finder for values don't have trailing os sep for folders
-      local path = file:absolute()
+      local path = newly_created_root(file, base_dir)
```

Comment:
Selection callback should only be created if path not nil. `path` is probably better named `selection_path` for clarity.

Consider adding a comment